src/TranslateExtension.php

```diff
@@ -384,4 +402,17 @@ protected function getDefaultConfig()
             'show_flags' => true
         ];
     }
+
+    /**
+     * Set Fields for field builder event.
+     */
+    public function onRequest(FieldBuilderEvent $event)
+    {
+        $labels = [];
+        $config = $this->getConfig();
+        foreach ($config['locales'] as $language) {
+            $labels[] = array($language['slug'].'label' => $language['slug'].'label');
```

Comment:
Just a few small things here: Could you change to the short array syntax and put some spacing between the concat operators? something like this: `[$language['slug'] . 'label' => $language['slug'] . 'label'];`